src/backend/src/players/players.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { BadRequestException, Body, Controller, Post } from '@nestjs/common';
+import { Player } from '../entities/player.entity';
+
+@Controller('players')
+export class PlayersController {
+  @Post()
+  async create(
+    @Body('userId') userId: number,
+    @Body('emotionId') emotionId: number,
+  ): Promise<number> {
+    if (!userId || !emotionId) {
+      throw new BadRequestException('Not all player fields are filled');
```

Comment:
Nit, but it's good to be specific so something like "Create Player requires emotionId and Playerid"